app/models/sample.rb

```diff
@@ -11,4 +11,12 @@ class Sample < ApplicationRecord
 
   validates :name, presence: true, length: { minimum: 3, maximum: 255 }
   validates :name, uniqueness: { scope: %i[name project_id] }
+
+  def self.ransackable_attributes(_auth_object = nil)
+    %w[id name created_at updated_at]
+  end
+
+  def self.ransackable_associations(_auth_object = nil)
+    %w[name]
```

Comment:
I'm learning ransack, but isn't name an attribute, not an association?

Reply:
I've switched it to project in 767377956c07734fd18058edd4b8dc9ac1af1da1

Reply:
I decided to remove project and just leave an empty array as we're not currently using any associations for sample. It doesn't seem like associations can be removed (or even just the array removed) as its required for ransack's functionality. 13e96af4051663268db7796af8000806a1e0b364